docs/detections/rules-prebuilt.asciidoc

```diff
@@ -0,0 +1,142 @@
+
+* You can't modify most settings on Elastic prebuilt rules. You can only edit <<rule-schedule, rule actions>> and <<add-exceptions, add exceptions>>. If you want to modify other settings on a prebuilt rule, you must first duplicate it, then make your changes to the duplicated rule. 
+====
+
+[float]
+[[load-prebuilt-rules]]
+=== Install and enable Elastic prebuilt rules
+
+. Go to *Manage* -> *Rules*. The badge next to *Add Elastic rules* shows the number of uninstalled prebuilt rules available. 
++
+[role="screenshot"]
+image::images/prebuilt-rules-add-badge.png[The Add Elastic Rules page]
+
+. Click *Add Elastic rules*, then do one of the following:
+* Install all available rules: Click *Install all*.
+* Install a single rule: Click *Install rule* for that rule.
+* Install multiple rules: Select the rules and click *Install _x_ selected rule(s)*.
++
+TIP: Use the search bar and *Tags* filter to find the rules you want to install. For example, filter by `OS: Windows` if your environment only includes Windows endpoints. For more on tag categories, refer to <<prebuilt-rule-tags>>.
++
+[role="screenshot"]
+image::images/prebuilt-rules-add.png[The Add Elastic Rules page]
+
+. Go back to the *Rules* page, then search or filter for any rules you want to start running, and do the following:
+
+* Enable a single rule: Turn on rule's the *Enabled* switch.
+* Enable multiple rules: Select the rules, then click *Bulk actions* -> *Enable*.
+
+Once you enable a rule, it starts running on its configured schedule. To confirm that it's running successfully, check its *Last response* status in the rules table, or open the rule's details page and check the <<rule-execution-logs, *Execution results*>> tab.
+
+[float]
+[[prebuilt-rule-tags]]
+=== Prebuilt rule tags
+
+Each prebuilt rule includes several tags identifying the rule's purpose, detection method, associated resources, and other information to help categorize your rules. These tags are category-value pairs, such as `OS: Windows` for a rule designed for Windows endpoints. Categories include:
+
+* `Data Source`: The application, cloud provider, data shipper, or Elastic integration providing data for the rule.
+* `Domain`: A general category of data source types (such as cloud, endpoint, or network).
+* `OS`: The host operating system, which could be considered another type of data source.
+* `Resources`: Additional rule resources such as investigation guides.
+* `Rule Type`: Identifies if the rule depends on specialized resources (such as machine learning jobs or threat intelligence indicators), or if it's a higher-order rule built from other rules' alerts.
+* `Tactic`: MITRE ATT&CK tactics that the rule addresses.
+* `Threat`: Specific threats the rule detects (such as Cobalt Strike or BPFDoor).
+* `Use Case`: The type of activity the rule detects and its purpose. Use cases include:
+** `Asset Visibility`: Detects changes to specified asset types.
```

Comment:
Seems that one use case is missing here: `Active Directory Monitoring`
![image](https://github.com/elastic/security-docs/assets/137783811/7fa2bf07-814f-4101-af37-43ed334b2f6d)